src/eva/utilities/config.py

```diff
@@ -36,20 +36,20 @@ def __init__(self, dict_or_yaml, logger):
 
     # ----------------------------------------------------------------------------------------------
 
-    def get(self, key, default=None):
+    def get(self, key, default=None, abort_on_failure=True):
 
         if default is None:
 
             if key in self.config:
 
                 return super().get(key)
 
-            else:
+            elif abort_on_failure:
 
                 self.logger.abort("Configuration does not have the key")
 
         else:
-
+            logger.info(f'Warning: key {key} not found, returning default {default}')
```

Comment:
I removed this in #98 as the message is not correct.
This will warn even if the key is found as it is currently written.

Reply:
Ok, I've deleted the output.